From 722cd0ddf595be7c7ef8f2eeb5bb9ccee9690187 Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Mon, 8 Jan 2018 11:41:34 -0600
Subject: [PATCH 1/4] use mklfft when available

Use the mklfft library when it is available, use fftpack when not available.
---
 numpy/fft/__init__.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/numpy/fft/__init__.py b/numpy/fft/__init__.py
index 72d61a7..7c735e6 100644
--- a/numpy/fft/__init__.py
+++ b/numpy/fft/__init__.py
@@ -3,7 +3,12 @@ from __future__ import division, absolute_import, print_function
 # To get sub-modules
 from .info import __doc__
 
-from .fftpack import *
+try:
+    from mklfft.fftpack import *
+    using_mklfft = True
+except ImportError:
+    from .fftpack import *
+    using_mklfft = False
 from .helper import *
 
 from numpy.testing import _numpy_tester
-- 
2.7.4

